### ISSUE TITLE
More beautiful meditation countdown

### ISSUE THREAD

**Issue by snan** [mudita/MuditaOS#3404]
The meditation time is on a proportional font so as the seconds tick down, the entire time stamp jitters. If instead each glyph was fixed in space only the relevant number would move. It could still be relatively proportional, i.e. a thin `:` and so on. This would look awesome on e-ink ♥

+ also add a 20 minut schedule ♥♥

**Comment by pholat** [comment]
Actually whole timer should jitter as i.e. 1 and 0 have different width and inter font parameters. Saying that - wouldn't at best be the use od monospaced font in this particular example?

**Comment by snan** [comment]
As long as it's only the last digit that's changing, it wouldn't jitter. And going from 50 to 49, for example, both digits are changing at the same time. It's never the case that a non LSD changes on its own.

**Comment by snan** [comment]
So a left-justified (a.k.a. "ragged right") timer doesn't jitter.

**Comment by pholat** [comment]
@snan non-monospaced fonts have two parameters that should differ for different glyph combinations: this is [kerning](https://www.wikiwand.com/en/Kerning) and glyph width. 
Consider glyphs for `1` and `2` - both have different widths. Now when you write 50 times in i.e. LibreOffice with different fonts these would have different widths:
![image](https://user-images.githubusercontent.com/6411862/149086319-6b958f51-e3b8-46bc-aaf3-616892dabc08.png)
This is why I said that the whole timer should jitter. I felt that you suggested forcibly "holding" each character (in range of glyphs) and that actually makes non-monospaced font behave as a monospaced one. This is why I suggested monospaced font. As it has the same width and no kerning for each character, it's just the simplest approach. 

Saing all of that you would be pretty happy if it didn't jitter :)

**Comment by mjan-dev** [comment]
@snan FYI, we are planning to introduce monospaced font for use with timers across the system. 
I'm closing this issue for now. Thanks for the suggestion :+1:

**Comment by snan** [comment]
Wonderful ♥
Aesthetic touches & polish ftw ♥♥♥

**Comment by snan** [comment]
> @snan I felt that you suggested forcibly "holding" each character (in range of glyphs) and that actually makes non-monospaced font behave as a monospaced one.

I missed this reply.

I know how widths and kerns work.

A monospaced font timer doesn't jitter, but neither does a ragged-right (a.k.a. left-aligned) proportional font timer. (Because a digit can only change when all of the digits to the right of it changes, that's just math.) My suggestion was to make the minutes and seconds both be separately left-aligned (a.k.a. ragged right) and keep the `:` character steady between them.

cha
![IMG_0638](https://user-images.githubusercontent.com/309417/157029776-91612ab9-29d6-4d1a-933c-fd324dc23050.jpeg)
